raiden/transfer/events.py

```diff
@@ -102,14 +129,40 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+    def to_dict(self) -> typing.Dict[str, typing.Any]:
+        result = {
+            'channel_identifier': self.channel_identifier,
+            'token_network_identifier': to_checksum_address(self.token_network_identifier),
+        }
+
+        if self.our_balance_proof is not None:
+            result['our_balance_proof'] = self.our_balance_proof
```

Comment:
@palango Can we just assign those values even if they're None? its better to have the returned dict with key => None than not having the key at all in the serialized JSON.

Reply:
Sure, if it (de)serialized properly.